framework/source/translation/ru.po

```diff
@@ -0,0 +1,294 @@
+# Russian
+# 
+msgid ""
+msgstr ""
+"Project-Id-Version: qooxdoo framework\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2008-05-19 10:11+0200\n"
+"PO-Revision-Date: 2021-01-28 17:53+0300\n"
+"Last-Translator: Дмитрий Золотов <zolotovdy@yandex.ru>\n"
+"Language-Team: none\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: Russian\n"
+"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
+
+msgid "%1 does not fit %2."
+msgstr "%1 не подходит %2."
+
+msgid "%1 is not a color! %2"
+msgstr "%1 не является цветом! %2"
+
+msgid "%1 is not a number."
+msgstr "%1 не является числом."
+
+msgid "%1 is not a string."
+msgstr "%1 не является строкой."
+
+msgid "%1 is not an url."
+msgstr "%1 не является допустимым URL."
+
+msgid "%1 is not in %2"
+msgstr "%1 не входит в %2"
+
+msgid "%1 is not in the range from [%2, %3]."
+msgstr "%1 не входит в диапазон [%2, %3]."
+
+msgid "'%1' is not an email address."
+msgstr "'%1' не является эл. почтовым адресом."
+
+msgid "Automatic"
+msgstr "Авто"
+
+msgid "Basic Colors"
+msgstr "Основные цвета"
+
+msgid "Cancel"
+msgstr "Отмена"
+
+msgid "Color Selector"
+msgstr "Выбор цвета"
+
+msgid "Details"
+msgstr "Подробно"
+
+msgid "HSB"
+msgstr ""
+
+msgid "Hex"
+msgstr ""
+
+msgid "Last month"
+msgstr "Прошлый месяц"
+
+msgid "Last year"
+msgstr "Прошлый год"
+
+msgid "Next month"
+msgstr "Следующий месяц"
+
+msgid "Next year"
+msgstr "Следующий год"
+
+msgid "OK"
+msgstr "OK"
+
+msgid "Open ColorSelector"
+msgstr "Открыть диалог Выбор цвета"
+
+msgid "Presets"
+msgstr "Наборы"
+
+msgid "Preview (Old/New)"
+msgstr "Предпросмотр (До/После)"
+
+msgid "RGB"
+msgstr ""
+
+msgid "Recent Colors"
+msgstr "Недавние"
+
+msgid "Reset column widths"
+msgstr "Сбросить длины столбцов"
+
+msgid "This field is required"
+msgstr "Заполните это поле"
+
+msgid "Visual"
+msgstr "Визуально"
+
+msgid "key_full_Alt"
+msgstr ""
```

Comment:
Is there no translation for these that were left blank? If that's the case, then maybe translate them to English rather than making them blank?

Reply:
I thought it assigns English by default. All blanks I left on purpose. Thank you, I will change it and commit it.